SQL/0000-00-00-schema.sql

```diff
@@ -10,7 +10,7 @@ CREATE TABLE `Project` (
     PRIMARY KEY (`ProjectID`)
 ) ENGINE = InnoDB  DEFAULT CHARSET=utf8;
 
-INSERT INTO `Project` (Name) VALUES ('loris');
+INSERT INTO `Project` (Name,Alias) VALUES ('loris','LORI');
```

Comment:
LORI? should it be LORIS

Reply:
its 4 char max...

Longitudinal Online Research and Imaging !! thats the project lol

Reply:
@ridz1208 oh lol.